src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1042,6 +1046,9 @@ internal override int GetSyntaxTreeOrdinal(SyntaxTree tree)
 
         internal OneOrMany<SyntaxTree> GetSyntaxTreesByMappedPath(string mappedPath)
         {
+            // This method supports a "compat" behavior for interceptor file path resolution.
+            // It must be removed prior to stable release.
```

Comment:
nit: consider adding link to `[https://github.com/dotnet/roslyn/issues/72265`](https://github.com/dotnet/roslyn/issues/72265`) here as well